twentytwenty-blocks/twentytwenty-styles/editor-style-block.css

```diff
@@ -249,29 +254,30 @@ Inter variable font. Usage:
 .editor-styles-wrapper .wp-block h6 {
 	font-feature-settings: "lnum";
 	font-variant-numeric: lining-nums;
-	font-weight: 700;
+	font-weight: var(--wp--typography--font-weight-heading);
```

Comment:
Have the `--wp--typography--` variables gone away? I don't see them being applied anymore when I activate this theme. Do they need to be converted to presets in the theme config?

Reply:
Ah, whoops. That's a leftover from the last attempt at this. It needs to be converted to use the new system.

Reply:
This should be take care of in fe1e88e.

Reply:
Actually wait — I need to get it fixed in the editor styles too. 👍

Reply:
Ok, should be ok as of a442c52.